File: src/main/java/org/zenbot/timetableupdater/batch/Timetable.java
package org.zenbot.timetableupdater.batch;

import lombok.Data;

import java.util.HashMap;
import java.util.Map;

@Data
public class Timetable {

    private String routeName;
    private String startBusStopName;
    private String endBusStopName;
    private String activeStopName;
    private Map<Integer, Map<String, String>> timetable = new HashMap<>();

    public void addRow(int hour, Map<String, String> values) {
        timetable.put(hour, values);
    }
}

File: src/main/kotlin/org/zenbot/timetableupdater/batch/Timetable.kt
package org.zenbot.timetableupdater.batch

data class Timetable (
        var routeName: String = "",
        var startBusStopName: String = "",
        var endBusStopName: String = "",
        var activeStopName: String = "",
        var timetable: MutableMap<Int, Map<String, String>> = mutableMapOf()
) {
    fun addRow(hour: Int,values: Map<String, String>) = timetable.put(hour, values)
}